package.json

```diff
@@ -105,8 +105,8 @@
     "lint-staged": "^9.2.2",
     "prettier": "^1.18.2",
     "prop-types": "^15.7.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
+    "react": "^16.9.0",
```

Comment:
We need to change these all at once, with the packages. Can you undo these please?